### DISCUSSION TITLE
Car features

### DISCUSSION THREAD

**Discussion by larronni** [flobz/psa_car_controller#913] [Q&A] [answered]
When I check "get_vehicleinfo" I get a list of attributes. But when the attribute has the value "null", does it mean that the car does not support this feature, or is it something wrong with my setup?
I have a DS3 and many of the features has value "null", among position:
```
  "last_position": {
    "type": "Feature",
    "geometry": {
      "coordinates": [null, null, null],
      "type": "Point"
    }
```

**Comment by GeraldPape** [comment, marked as answer]
It means there is no value present. Your setup seems OK.
I never got those values, until I registered the app EEVEE on my mobile and allowed it access to my car data. 
For me it seems like PSA is only sending the data, when there is an application requesting it.
